.github/workflows/build.yml

```diff
@@ -20,12 +20,14 @@ jobs:
         with:
           fetch-depth: 0
       - name: Install pip dependencies
-        run: pip install -r requirements.txt
+        run: |
+          pip install -r requirements.txt
+          pip install -e .
```

Comment:
Future proofing?

Reply:
In theory this is necessary for the `from pyteal import __all__ as static_all` line to work in `scripts/generate_init.py`, but I haven't tested without it

Reply:
👍